generic/java/egg-java.json

```diff
@@ -1,19 +1,24 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2021-07-21T18:23:17+03:00",
+    "exported_at": "2022-10-24T19:20:23+02:00",
     "name": "Generic Java",
     "author": "sneaky@sneakyhub.com",
     "description": "Creates a container that runs java.",
     "features": null,
-    "images": [
-        "ghcr.io\/parkervcp\/yolks:java_8",
-        "ghcr.io\/parkervcp\/yolks:java_11",
-        "ghcr.io\/parkervcp\/yolks:java_16"
-    ],
+    "docker_images": {
```

Comment:
only 8,11,16,17,19 are sttill build: https://github.com/parkervcp/yolks/blob/master/.github/workflows/java.yml

Reply:
> only 8,11,16,17,19 are sttill build: [parkervcp/yolks@`master`/.github/workflows/java.yml](https://github.com/parkervcp/yolks/blob/master/.github/workflows/java.yml?rgh-link-date=2022-10-24T17%3A31%3A42Z)

Then someone should remove the old versions from the readme